<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnTech::GuiQt::MetaSprite::MetaSprite::PalettesDock</class>
 <widget class="QDockWidget" name="UnTech::GuiQt::MetaSprite::MetaSprite::PalettesDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>207</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Palettes</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QListView" name="paletteList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>80</height>
       </size>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolBar" name="paletteListButtons">
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="floatable">
       <bool>false</bool>
      </property>
      <addaction name="action_Add"/>
      <addaction name="action_Clone"/>
      <addaction name="action_Raise"/>
      <addaction name="action_Lower"/>
      <addaction name="action_Remove"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="toolButtonsLayout">
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="editColorButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="selectColorButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Select</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="selectedPalette">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="colorGrid">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action_Add">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Palette</string>
   </property>
  </action>
  <action name="action_Clone">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/clone.svg</normaloff>:/icons/clone.svg</iconset>
   </property>
   <property name="text">
    <string>Clone Palette</string>
   </property>
  </action>
  <action name="action_Raise">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/raise.svg</normaloff>:/icons/raise.svg</iconset>
   </property>
   <property name="text">
    <string>Raise Palette</string>
   </property>
  </action>
  <action name="action_Lower">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/lower.svg</normaloff>:/icons/lower.svg</iconset>
   </property>
   <property name="text">
    <string>Lower Palette</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/remove.svg</normaloff>:/icons/remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Palette</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
